docs/source/Plugin/P146.rst

```diff
@@ -103,6 +109,111 @@ Commands available
 
 .. .. include:: P146_events.repl
 
+Example
+-------
+
+Planned use for below Rules is a set of sensors and a GPS connected to an ESP(32), mounted in a RV (Camper). During the ride, the GPS generates an event at configured distance intervals, causing the data to be stored in the Cache Controller, including time-stamp by using the Dummy Device that will include that when sending data.
```

Comment:
Using a dummy task allows to combine some values from separate sensors and thus allow to store it slightly more compact.
Also the command ``TaskRunAt`` allows for dummy tasks to flush their data to the Cache Controller to have the same timestamp. Thus these values can later be combined by the CacheReader task on a single CSV line making it easier to process the data recorded during the trip.